apm-server/README.md

```diff
@@ -8,8 +8,8 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
-**Warning**: This branch is used for development, please use [7.7.1][] release
-for released version.
+**Warning**: This branch is used for development, please use the latest [7.10](https://github.com/elastic/helm-charts/releases)
```

Comment:
7.10 isn't released yet so this is not accurate. Also this would require us to update it each time we would release a new minor version (7.11, ...) to stay up to date.

Maybe we could just mention the latest [7.x](https://github.com/elastic/helm-charts/releases) release

Reply:
same comment about having the warning statement in a single line